package/ubuntu/hypercontainer/make-hypercontainer-deb.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+PROJECT=$(readlink -f $(dirname $0)/../../..)
+UBUNTU_DIR=${PROJECT}/package/ubuntu/hypercontainer
+VERSION=0.6.2
+
+if [ $# -gt 0 ] ; then
+    VERSION=$1
+fi
+
+# install addtional pkgs in order to build deb pkg
+sudo apt-get install -y autoconf automake pkg-config libdevmapper-dev libsqlite3-dev libvirt-dev qemu libvirt-bin -qq
```

Comment:
I think you could list these packages in Build Dependencies of `control` meta file:

`libdevmapper-dev libsqlite3-dev libvirt-dev`

And I think this two file is not required for build procedure: ` qemu libvirt-bin`